Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -182,12 +182,22 @@ def GetPlatformDsc (self):
 
     def GetKeyHashList (self):
         # Define a set of new key used for different purposes
-        # The key is either public key PEM format or private key PEM format
+        # The key is either key id or public key PEM format or private key PEM format
         pub_key_list = [
           (
-            # Use a single test key
-            HASH_USAGE['PUBKEY_CFG_DATA'] | HASH_USAGE['PUBKEY_FWU'] | HASH_USAGE['PUBKEY_OS'] | HASH_USAGE['PUBKEY_CONT_DEF'],
-            'TestSigningPrivateKey.pem'
+            # Use a single test key Id CFGDATA_KEY_ID

```

Comment:
Update these comments accordingly:
Key for verifying Config data blob.

Reply:
Updated descriptions.